README.md

```diff
@@ -25,28 +29,33 @@ The `.deb` that Tauri builds will automatically do this for us for actual toolki
 
 # 🔧 Setup
 
-1. (New Method) Run the following command.
+To install Deakin Detonator Toolkit on Kali, you can follow either the new or old methods. The new method is a one&#8209;step process that utilises a Python script. If that doesn't work, the old method will take you through the steps manually. 
```

Comment:
To install the Deakin Detonator Toolkit

Reply:
https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/pull/489/commits/eeb11e0cfe614a7a9ba65bd02e359ff023c7670c